exercises/performance/multi-threaded-server/build.gradle

```diff
@@ -40,7 +41,7 @@ if (JavaVersion.current().toString() != javaVersionTarget) {
 }
 
 application {
-    mainClass = 'App'
```

Comment:
Can you rename the SingleThreadedServer instead? We should try to keep build.gradle files as close to each other as possible. Same remark for the others.

Reply:
Yes I could but as they are supposed to change it while doing the exercise, we should find something more consistent. IMHO having `App` and then `AppMultiThreaded` does not make a lot of sense. Do you have other name ideas?